sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -1098,3 +1098,36 @@ def is_consistent_with(sub, base):
     # Nothing but object lives above any type constraints.
     return base == object
   return issubclass(sub, base)
+
+
+def coerce_to_kv_type(element_type, label=None):
```

Comment:
<!--new_thread; commit:e51bdb947d47ba6744a5b19c72ba0d8fa1c06045; resolved:0-->
Do the branches here currently have any test coverage?